packages/flutter_tools/templates/plugin_swift_package_manager/ios-swift.tmpl/projectName.tmpl/Package.swift.tmpl

```diff
@@ -17,7 +17,14 @@ let package = Package(
             name: "{{projectName}}",
             dependencies: [],
             resources: [
-               .process("Resources"),
+                // If your plugin requires a privacy manifest, for example if it uses any required
+                // reason APIs, update the PrivacyInfo.xcprivacy file to describe your plugin's
+                // privacy impact, and then uncomment these lines. For more information, see
+                // https://developer.apple.com/documentation/bundleresources/privacy_manifest_files
+                // .process("PrivacyInfo.xcprivacy"),
+
+                // If you have other resources that need to be bundled with your plugin, add the
+                // files or enclosing directories as new .process(...) entries.
```

Comment:
This is fine with me, maybe link to https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package? Not all resources have to be explicitly defined

```suggestion
                // If you have other resources that need to be bundled with your plugin, refer to the 
                // following instructions to add: https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package
```

Reply:
Done